ui/js/graphs.js

```diff
@@ -430,15 +443,15 @@ perf.controller('GraphsCtrl', [
         series: $scope.seriesList.map(function(series) {
           return encodeURIComponent(
             JSON.stringify(
-              { project: series.projectName,
-                signature: series.signature,
-                visible: series.visible})); }),
+              [ series.projectName,
+                series.signature,
+                series.visible])); }),
         highlightedRevisions: _.filter($scope.highlightedRevisions,
                                        function(highlight) {
                                          return (highlight &&
                                                  highlight.length == 12);
                                        }),
-        zoom: JSON.stringify($scope.zoom)
+        zoom: encodeURIComponent(JSON.stringify(zoomStringify()))
```

Comment:
Can we avoid the need for JSON.stringify and encodeURIComponent here? I don't think we should strictly need to do this, and it bloats the size of the URL